README.md

```diff
@@ -11,7 +11,7 @@ Key Features:
 
 Feel free to explore the Exasol AI-Lab and unleash your data science potential!
 
-The AI-Lab is available in multiple [Editions](doc/user_guide/editions.md) involving different technology stacks, see also common [System Requirements](doc/user_guide/system-requirements.md).
+The AI-Lab is available in multiple [Editions](blob/2.0.0/doc/user_guide/editions.md) involving different technology stacks, see also common [System Requirements](doc/user_guide/system-requirements.md).
```

Comment:
This hard-codes the version. Is this intended? Maybe also update the paths to the other .md files.

Reply:
The hard-coded link to tag 2.0.0 is deliberate. 
I updated the link to `jupyter.md`, too. 

The other links are lower priority, as pointing to developer guide and an image, whereas our major concern is users we want to point to the latest released version of the documentation. 

I also wanted to keep the impact small. Additionally, the links need to be excluded from link checker and cannot be resolved locally. So all in all, this is a poor-man's versioning. :slightly_frowning_face: